go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/grafana/grafana-iot-twinmaker-app
 go 1.16
```

Comment:
should we change this to 1.18?

the deployment is with 1.19 -- so should be safe (though not necessary)

Reply:
makes sense to me to bump it to 1.18. updated in [dc5489d](https://github.com/grafana/grafana-iot-twinmaker-app/pull/121/commits/dc5489d2c15dc2c590fa7f5e174bee7087eac9e7), but can revert if it's an issue on the AWS side

Reply:
If we update this would that lead to us reorganizing GoLang code with the new features (generics, etc.)?
Or will upgrading will resolve security risks?
If this is a plugin best practice then we can upgrade. 
Only risk is if our code isn't backwards compatible with the dependency upgrade.

Reply:
> If we update this would that lead to us reorganizing GoLang code with the new features (generics, etc.)?
>  Or will upgrading will resolve security risks?

it sets the minimum compatible go version, which in this case gives us access to features added in 1.17 & 1.18. since the plugin is currently built with go 1.19 in CI, it should be safe to require 1.18

>  Only risk is if our code isn't backwards compatible with the dependency upgrade.

not sure what you mean here. i think the plugin wouldn't build if there were compatibility issues